app/authentication/views.py

```diff
@@ -27,7 +27,7 @@ def login(request):
     user = _try_to_get_user(user_id=user_id)
 
     if user.check_password(password):
-        if user.is_TIHLDE_member:
+        if user.memberships.filter(group__slug=Groups.TIHLDE).exists():
```

Comment:
Du kan bare beholde propertien siden det ikke var den som var problemet

Reply:
Spørsmålet er om vi har bruk for den? Brukes ikke på frontend i det hele tatt

Reply:
Nei, men det er mye cleanere hvis den ligger på User som en property, slik som det var før. Du trenger ikke å legge den på serializeren

Reply:
Ahh, godt poeng! Legger tilbake 👍